canvas_account_admin_tools/settings/local.py

```diff
@@ -57,6 +58,13 @@
          'type': 'All courses',
          'year': '2016',
       },
+      # TLT-2574 - Course Info People Add
+      'test_data_for_multiple_users_add': {
+         'canvas_role': 'Guest',
+         'unsuccessful_add': '12345678, 23456789',
```

Comment:
No no! I misread it. :) I though it read `'x', 'y'` instead of `'x, y'`. My bad, thanks for the clarification!